packages/mcp/mcp-client-chat/src/mcp-client-chat.ts

```
@@ -21,6 +21,8 @@ export class McpClientChat {
   protected clientsMap: Map<string, Client> = new Map();
   protected toolClientMap: Map<string, Client> = new Map();
   protected messages: Message[];
+  protected transformStream: TransformStream = new TransformStream();
```

ESLint

[error] The 'TransformStream' is still an experimental feature The configured version range is '^20.19.0 || >=22.12.0'.

(n/no-unsupported-features/node-builtins)